File: 240625/정수만 추출하기/extract-only-integers.py
first, second = input().split()

first_int, second_int = 0, 0
first_str, second_str = "", ""
for f_str in first:
    if not f_str.isdigit():
        first_int += int(first_str)
        break
    else:
        first_str += f_str

for s_str in second:
    if not s_str.isdigit():
        second_int += int(second_str)
        break
    else:
        second_str += s_str

print(first_int + second_int)

File: 251211/정수만 추출하기/extract-only-integers.cpp
#include <iostream>
#include <string>
#include <cctype>
using namespace std;

int main() 
{
    string first, second;
    cin >> first >> second;

    string first_str = "", second_str = "";

    for(char c : first) 
    {
        if(!isdigit(c)) break;
        first_str += c;
    }

    for(char c : second) 
    {
        if(!isdigit(c)) break;
        second_str += c;
    }

    int first_int = first_str.empty() ? 0 : stoi(first_str);
    int second_int = second_str.empty() ? 0 : stoi(second_str);

    cout << first_int + second_int;

    return 0;
}
